src/Symfony/Component/Form/Extension/Core/DataAccessor/CallbackAccessor.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Extension\Core\DataAccessor;
+
+use Symfony\Component\Form\DataAccessorInterface;
+use Symfony\Component\Form\Exception\RuntimeException;
+use Symfony\Component\Form\FormInterface;
+
+/**
+ * Writes and reads values to/from an object or array using callback functions.
+ *
+ * @author Yonel Ceruto <yonelceruto@gmail.com>
+ */
+class CallbackAccessor implements DataAccessorInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getValue($data, FormInterface $form)
+    {
+        if (null !== $getter = $form->getConfig()->getOption('get')) {
+            return ($getter)($data, $form);
+        }
+
+        throw new RuntimeException('Unable to read the given form data.');
```

Comment:
Maybe we could introduce a new `AccessException` to make this a little more specific? Not sure what the policy is on specific exception classes.

Reply:
Good idea! I will create the new exception class and update the interface. Thanks!